help/de/Help_ch_Account-Actions.xml

```diff
@@ -280,115 +279,37 @@
           </row>
 
           <row>
-            <entry namest="c1">
+            <entry>
               <para>Investmentfonds
               </para>
             </entry>
 
-            <entry namest="c2">
-              <para>Ein professionell verwaltetes Portfolio aus Aktien, Anleihen oder anderen Geldanlagen, die sich in
-                Anteile teilen lassen.
+            <entry>
+              <para>Ein professionell verwaltetes Portfolio aus Aktien, Anleihen oder anderen Geldanlagen, das sich in
+                Anteile teilen läßt.
               </para>
             </entry>
           </row>
 
           <row>
-            <entry namest="c1">
+            <entry>
               <para>Aktienkonto
               </para>
             </entry>
 
-            <entry namest="c2">
-              <para>Aktien oder Anteile an einer Gesellschaft, die den Eigentümer an all ihren Gewinnen und Risiken
-                beteiligen.
+            <entry>
+              <para>Aktien oder Anteile an einer Gesellschaft, die ihre Eigentümer an all ihren Gewinnen und Verlusten
+                beteiligt.
```

Comment:
Vielleicht besser (mit BP): Aktien oder ähnliche Eigenkapital-Anteile.

Ob man den Rest noch irgendwo erklären muß? Vielleich ganz knapp mit einem Verweis auf eine ausführliche Erklärung im Guide.

Zum grundsätzlichen Verständnis: Du kannst dich am 1. Eigenkapital oder 2, Fremdkapital einer Firma beteiligen.

1. Du kaufst 10 Aktien/Gesellschaftsanteile á 100 € und hoffst, daß die Firma ordentlich Gewinn macht. 
2. Du kaufst –nach dem Motto "Lieber den Spatz in der Hand …"– 10 Bonds/Schuldverschreibungen der Firma Laufzeit 10 Jahre, Jahreszinssatz 10% je nach Vertrag ausschüttend, aufgezinst oder sogar als "Finanzinnovation" abgezinst.

Szenario: Laufen die Geschäfte …

1.  super, steigt der Wert der Aktie um 100% –ggf. abzgl. Dividendenzahlung– während der Anleihenkäufer –wie vereinbart– nur 10% bekommt.
2. so lala, bekommt der Anleihenkäufer –wie vereinbart– 10% und der Aktionär geht leer aus.
3. worst case (Insolvenz), bekommt der Anleihenkäufer nur einen Bruchteil seiner Einlage zurück, während der Aktionär wahscheinlich das ganze Investment abschreiben muß. Bei Gesellschaftsformen ohne Haftungsbegrenzung (GBR, OHG; …) muß der Anteilseigner die Ansprüche der Gläubiger auch noch aus seinem Privatvermögen befriedigen.

Reply:
@fellen: Hier stehe ich, trotz deiner guten Erläuterung, auf dem Schlauch. Kannst du einen Text formulieren?

Reply:
Die Aktie (englisch share) ist ein Wertpapier, das den Anteil an einer Aktiengesellschaft oder einer Kommanditgesellschaft auf Aktien (KGaA) verbrieft.`<footnote><ulink "&url-wp-de;Aktie"/></footnote>`

Wenn einem selbst nichts Vernünftiges eingfällt… ;-)

Reply:
ok.